snappy-internal.h

```diff
@@ -36,13 +36,44 @@
 namespace snappy {
 namespace internal {
 
+#if !defined(SNAPPY_HAVE_BMI2)
```

Comment:
Thank you for realizing we need these #defines and #includes in the header, for the `V128` type alias.

Can you please move this block outside namespaces?

I think that #including C system headers in a C++ namespace is undefined behavior.

Reply:
Thanks for pointing this out, will update.